README.md

```diff
@@ -47,45 +35,30 @@ As a customer
-
-* A free account on Twilio will only allow you to send texts to "verified" numbers. Use your mobile phone number, don't worry about the customer's mobile phone.
-* Finally submit a pull request before Monday at 9am with your solution or partial solution.  However much or little amount of code you wrote please please please submit a pull request before Monday at 9am
+To tackle the above task, I took the user stories above and created a domain model which set out 4 classes: Takeaway, Menu, Order and TextConfirmation. Takeaway delegated methods and tasks to each of the other classes to fulfil the user stories.
 
+The user stories are tested in the feature_spec.rb.
 
-In code review we'll be hoping to see:
-
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc. 
-
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance will make the challenge somewhat easier.  You should be the judge of how much challenge you want this weekend.
-
-Notes on Test Coverage
-------------------
-
-You can see your [test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) when you submit a pull request, and you can also get a summary locally by running:
+## How to run
 
 ```
-$ coveralls report
+2.3.1 :002 > takeaway = Takeaway.new
+ => #<Takeaway:0x007fef0421ed50 @menu=#<Menu:0x007fef0421ed28>, @order=#<Order:0x007fef0421ed00 @menu=#<Menu:0x007fef0421ecd8>, @current_order=[], @bill=0, @string_summary="">, @text_confirmation=#<TextConfirmation:0x007fef0421ec38>>
+2.3.1 :003 > takeaway.read_menu
+ => {"Spring rolls"=>2.5, "Pho"=>7.0, "Tofu stirfry"=>8.5}
+2.3.1 :004 > takeaway.make_selection('Spring rolls')
+ => "Order received for 1 Spring rolls."
+2.3.1 :005 > takeaway.make_selection('Tofu stirfry', 3)
+ => "Order received for 3 Tofu stirfry."
+2.3.1 :006 > takeaway.order_summary
+ => " Spring rolls x 1: £2.50 | Tofu stirfry x 3: £25.50 |"
+2.3.1 :007 > takeaway.place_order
 ```
 
-This repo works with [Coveralls](https://coveralls.io/) to calculate test coverage statistics on each pull request.
-
-Build Badge Example
-------------------
+## Limitations/Improvements
 
-[![Build Status](https://travis-ci.org/makersacademy/takeaway-challenge.svg?branch=master)](https://travis-ci.org/makersacademy/takeaway-challenge)
-[![Coverage Status](https://coveralls.io/repos/makersacademy/takeaway-challenge/badge.png)](https://coveralls.io/r/makersacademy/takeaway-challenge)
+- I would implement greater testing of behaviours, rather than states.
+- I would make improvements to checking the bill against running total - currently there is an error raised if the bill does not equal total price of items. However, there is no way for the customer to check the bill against their own calculations.
```

Comment:
I feel this user story is a little ambiguous.  I think fine to just have a method that reports the total for the order.  Then the user checks it against a manual total they may have come to in their head.